contracts/TokenRulesMock.sol

```diff
@@ -21,4 +39,21 @@ contract TokenRulesMock {
         allowedTransfers[msg.sender] = false;
     }
 
+    function executeTransfers(
+        address _from,
+        address[] _transfersTo,
+        uint256[] _transfersAmount
+    )
+        external
+        onlyRule
```

Comment:
Do we need to keep `onlyRule` ?

Reply:
It was done to keep executeTransfers method same as it is in token rules.